packages/lesswrong/lib/collections/votes/schema.ts

```diff
@@ -42,12 +42,15 @@ const schema: SchemaType<DbVote> = {
     canRead: [userOwns, docIsTagRel, 'admins'],
     foreignKey: 'Users',
   },
-  
-  // The ID of the author of the document that was voted on
-  authorId: {
-    type: String,
+
+  // The IDs of the authors of the document that was voted on
+  authorIds: {
```

Comment:
I feel a bit uncomfortable at removing this. It will definitely break some analytics stuff for us, which is probably okay, but I have a feeling it will cause us headaches and that our general paradigm assumes documents have one canonical owner. One example, I assume revisions are still tied to one `userId` and am not yet sure how this PR will interact with that. (But only partway through reading it)

(I do separately think there's an anti-pattern around having both `thing` and `thing-plural` and that the existence of both `email` and `emails` has hurt us. (My ideal there is to get rid of the latter).

Reply:
This is a good point that I hadn't thought of. Whereabouts is it exposed in the API - I still can't find it? (Also, FWIW, a case-insensitive search for `authorid` in the GreaterWrong codebase returns 0 results).

For now, I'll go ahead and add a resolver-only `authorId` as this seems to fix the analytics issue (albeit, not beautifully). I feel that this field should generally be considered deprecated though. If there's any reason why you think this won't be enough then let me know and I'll give it a deeper think.

RE documents generally have only one canonical owner: I think this doesn't really matter here as semantically the "owner" of a vote is the `userId` who cast the vote, not the authors receiving the karma. I definitely could be missing something here though so feel free to object.